packages/modules/src/modules/banners/ausBrandMoment/AusBrandMomentBanner.tsx

```diff
@@ -0,0 +1,306 @@
+import React, { useState, useEffect, useRef } from 'react';
+import { css } from '@emotion/react';
+import { neutral, space } from '@guardian/src-foundations';
+import { Container, Hide } from '@guardian/src-layout';
+import { BannerRenderProps } from '../common/types';
+import { AusBrandMomentBannerHeader } from './components/AusBrandMomentBannerHeader';
+import { AusBrandMomentBannerArticleCount } from './components/AusBrandMomentBannerArticleCount';
+import { AusBrandMomentBannerBody } from './components/AusBrandMomentBannerBody';
+import { AusBrandMomentBannerCtas } from './components/AusBrandMomentBannerCtas';
+import { AusBrandMomentBannerCloseButton } from './components/AusBrandMomentBannerCloseButton';
+import { AusBrandMomentBannerVisual } from './components/AusBrandMomentBannerVisual';
+import { BannerTemplateSettings } from './settings';
+import { from } from '@guardian/src-foundations/mq';
+import { SecondaryCtaType } from '@sdc/shared/types';
+import { AusBrandMomentBannerReminder } from './components/AusBrandMomentBannerReminder';
+
+// ---- Banner ---- //
+
+export function getAusBrandMomentBanner(
+    templateSettings: BannerTemplateSettings,
+): React.FC<BannerRenderProps> {
```

Comment:
I wonder if we can just use the existing `getMomentTemplateBanner` function instead of creating this new one?

If so, we can just import it here and then move that `GreyBanner` code from the stories file into here but rename it to `AusBrandMomentBanner`. For the remote module importing to work on dotcom, we need the file containing the banner (i.e this file!) to export it with the same name as the file (i.e `AusBrandMomentBanner`!)

Reply:
Here's an example of how we used it for the one of the post election Aus banners: https://github.com/guardian/support-dotcom-components/blob/main/packages/modules/src/modules/banners/postElectionAuMoment/PostElectionAuMomentAlbaneseBanner.tsx

Reply:
Oh that's hilarious. I totally misunderstood what template meant! Thanks for redirecting me so kindly.

Reply:
I'll open a new PR with the correct approach - and way less code! 👏🏻